app.py

```diff
@@ -153,12 +151,18 @@ def generate_select_boxes(options_x, options_y, options_color, key_prefix):
             )
             bins = st.slider(label="Bins", min_value=1, max_value=100, value=10)
             ordinal = st.checkbox(label="Ordinal", value=False)
+            normalize = st.checkbox(label="Normalize", value=False)
 
             if not col_x:
                 st.warning("Please select a value for X.")
             elif not col_y and not col_color:
-                st.subheader("Simple histogram")
-                plot_histo(df, col_x=col_x, bin=bins, ordinal=ordinal)
+                if ordinal and normalize:
+                    st.warning("Please select only one (Ordinal or Normalize)")
```

Comment:
Ideally, we should aim to have both ordinal and normalize checkable at the same time. I can take a look at it, doesn't seem very simple at first glance.

Reply:
Hey! can you help me with this? I am not able to figure out how to do this

Reply:
Sure, I will take a closer look at it.